guides/slack-guide.mdx

```diff
@@ -36,6 +36,10 @@ Creating a Slack app is fairly simple. Follow these steps to create your app:
 <Info>Whatever is placed inside double braces is a variable.</Info>
 5. Make sure that you've turned on the Slack integration in the [Integrations Store](https://web.novu.co/integrations).
 <Frame caption="Make sure that Slack integration is set to active"><img src="/images/guides/slack-guide/integration.png" /> </Frame>
+6. To turn the Slack integration on, you'll need `Client Id` from the [Slack Developer's Dashboard](https://api.slack.com/apps). 
+<Frame caption="Get 'Client Id' from the Slack developer's dashboard"><img src="/images/guides/slack-guide/client-id-Slack.png" /> </Frame>
+7. Once you have it, you need to plug it into the respective field in the [Slack Integration Settings](https://web.novu.co/integrations) on the Novu Web Dashboard.
+<Frame caption="Plug the 'Client Id' into the Slack Integration setting in Novu web dashboard"><img src="/images/guides/slack-guide/client-id-Novu.png" /> </Frame>
```

Comment:
I think you also need to add these steps at the beginning when the user is creating the Slack app. Something around letting the reader know that they should keep the credentials safe because they will be needing it later in the guide.

Reply:
Addressed ✅